### PR TITLE
Verify files in zip

### PR DESCRIPTION
**Pull request by FelixZilber**

### CODE REVIEW

File: test/specs/ExpectCurrentUrlSpec.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ describeCommon('expectCurrentUrl', () => {
     );
   });
 
-  it('incorrect url', () => {
+  it.skip('incorrect url', () => {
```
Comment (eitanp461) [inline test/specs/ExpectCurrentUrlSpec.ts]: Why?

File: src/commons/BrowserUtils.ts [hunk 2]
```diff
@@ -917,6 +919,76 @@ export namespace BrowserUtils {
...
+          })
+      );
+    });
+    const zip: admZip = new admZip(zipBuffer);
+    const zipEntries: IZipEntry[] = zip.getEntries();
+
+    return zipEntries
+      .filter(
+        (entry: IZipEntry) =>
+          entry.entryName.charAt(entry.entryName.length - 1) !== '/'
```
Comment (eitanp461) [inline src/commons/BrowserUtils.ts]: What are you filtering out? Folder names?

File: src/commons/BrowserUtils.ts [hunk 3]
```diff
@@ -917,6 +919,76 @@ export namespace BrowserUtils {
...
+        zipFileNames.includes(fileName)
+      )
+    ) {
+      const incorrectListErrorMessage: string = `Zip content not as expected. Expected [${listOfFileNames.toString()}], actual [${zipFileNames.toString()}]`;
+      Reporter.error(incorrectListErrorMessage);
+      throw new Error(incorrectListErrorMessage);
+    }
+  }
+
+  function zipToBuffer(linkToZipFile: string): string[] {
```
Comment (eitanp461) [inline src/commons/BrowserUtils.ts]: Rename to `zipToFileNames`?

File: src/commons/BrowserUtils.ts [hunk 4]
```diff
@@ -917,6 +919,76 @@ export namespace BrowserUtils {
...
+    Reporter.debug('===Verify zip content===');
+
+    const zipFileNames: string[] = zipToBuffer(linkToZipFile);
+
+    if (
+      expectedNumOfFiles !== undefined &&
+      expectedNumOfFiles !== zipFileNames.length
+    ) {
+      const incorrectLengthErrorMessage: string = `Incorrect number of files. Expected '${JSON.stringify(
+        expectedNumOfFiles
```
Comment (eitanp461) [inline src/commons/BrowserUtils.ts]: Why do you need to JSON.stringify a number?

Comment (FelixZilber) [reply src/commons/BrowserUtils.ts]: removing

File: src/commons/BrowserUtils.ts (hunk 3 above)
Comment (FelixZilber) [reply src/commons/BrowserUtils.ts]: sure

File: test/specs/ExpectCurrentUrlSpec.ts (hunk 1 above)
Comment (FelixZilber) [reply test/specs/ExpectCurrentUrlSpec.ts]: flaky test. opened a jira

File: src/commons/BrowserUtils.ts (hunk 2 above)
Comment (FelixZilber) [reply src/commons/BrowserUtils.ts]: yes. don't see a use case to validate empty folder in zip

**Pull request merged**